KRKNWK-19540: Update RS/FEM projects related documentation (#17845)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/fem/21540ek_dev_guide.rst

.. parsed-literal::
   :class: highlight

   west build -b nrf5340dk/nrf5340/cpuapp -- -DSHIELD=nrf21540ek_fwd -Dipc_radio_SHIELD=nrf21540ek

> In this command, the *childImageName_* parameter has the ``ipc_radio_`` value as default and builds a application with support for combination of 802.15.4 and Bluetooth.
> The ``ipc_radio`` is used after build system was migrated to sysbuild. See :ref:`_child_parent_to_sysbuild_migration_sysbuild_configuration_file`
> Setting correct sysbuild option enables support for 802.15.4 and Bluetooth :ref:`ipc_radio`.

Reviewer: ```suggestion
In this command, the *childImageName_* parameter has the ``ipc_radio_`` value as default and builds the application with support for the combination of 802.15.4 and Bluetooth.
The ``ipc_radio`` has been used since the build system migration to sysbuild.
See :ref:`_child_parent_to_sysbuild_migration_sysbuild_configuration_file`.
Setting the correct sysbuild option enables support for 802.15.4 and Bluetooth :ref:`ipc_radio`.
```

*  ``ipc_radio`` represent all applications having support for combination of both 802.15.4 and Bluetooth. Application can be configured base on sysbuild configs:
      * ``SB_CONFIG_NETCORE_IPC_RADIO=y`` for applications having support for 802.15.4, but not for Bluetooth.
      * ``SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC=y`` for application having support for Bluetooth, but not for 802.15.4.

>       * ``SB_CONFIG_NETCORE_IPC_RADIO=y`` and ``SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC=y`` for multiprotocol applications having support for both 802.15.4 and Bluetooth.

Reviewer: ```suggestion
``ipc_radio`` represents all applications with support for the combination of both 802.15.4 and Bluetooth.
You can configure your application using the following sysbuild configurations:

* ``SB_CONFIG_NETCORE_IPC_RADIO=y`` for applications with support for 802.15.4, but not for Bluetooth.
* ``SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC=y`` for applications with support for Bluetooth, but not for 802.15.4.
* ``SB_CONFIG_NETCORE_IPC_RADIO=y`` and ``SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC=y`` for multiprotocol applications with support for both 802.15.4 and Bluetooth.
```

.. parsed-literal::
   :class: highlight

>    west build -b nrf5340dk/nrf5340/cpuapp -- -DSHIELD=nrf21540ek_fwd -Dipc_radio_SHIELD=nrf21540ek

Reviewer 2: do we put `--sysbuild` option ?

Reviewer 3: No: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/config_and_build/config_and_build_system.html#sysbuild-enabled-ncs

Author: It is not needed, sysbuild is enabled by default

> In this command, the *childImageName_* parameter has the ``ipc_radio_`` value as default and builds the application with support for the combination of 802.15.4 and Bluetooth.

Reviewer 3: AFAIK, child images are going to be removed in 2.9.0 -- any reason we still use this feature here?
https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/config_and_build/multi_image.html

Reviewer 2: I think this `_childImageName` is just a name for second application for the network/radio core here. Id does not refer to the "child image" feature that is deprecated. What better name to pick ?

The ``ipc_radio`` has been used since the build system migration to sysbuild.

> See :ref:`Migrating to sysbuild <child_parent_to_sysbuild_migration>` userguide.

Reviewer 3: ```suggestion
See :ref:`Migrating to sysbuild <child_parent_to_sysbuild_migration>` page.
```
